runtime-modules/project-token/Cargo.toml

```diff
@@ -5,28 +5,35 @@ authors = ['Joystream contributors']
-frame-system = { package = 'frame-system', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e'}
-sp-arithmetic = { package = 'sp-arithmetic', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e'}
-common = { package = 'pallet-common', default-features = false, path = '../common'}
-storage = { package = 'pallet-storage', default-features = false, path = '../storage'}
-balances = { package = 'pallet-balances', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e'}
-membership = { package = 'pallet-membership', default-features = false, path = '../membership'}
-codec = { package = 'parity-scale-codec', version = '3.1.2', default-features = false, features = ['derive'] }
-serde = {version = '1.0.101', features = ['derive'], optional = true}
-scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+sp-io = { package = 'sp-io', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+sp-std = { package = 'sp-std', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+sp-runtime = { package = 'sp-runtime', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+frame-support = { package = 'frame-support', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+frame-system = { package = 'frame-system', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+sp-arithmetic = { package = 'sp-arithmetic', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+common = { package = 'pallet-common', default-features = false, path = '../common' }
+storage = { package = 'pallet-storage', default-features = false, path = '../storage' }
+balances = { package = 'pallet-balances', default-features = false, git = 'https://github.com/joystream/substrate.git', rev = '50cf239147a6f569e563bcadec6c7a1c5ad5c67e' }
+membership = { package = 'pallet-membership', default-features = false, path = '../membership' }
+codec = { package = 'parity-scale-codec', version = '3.1.2', default-features = false, features = [
+	'derive',
+] }
+serde = { version = '1.0.101', features = ['derive'], optional = true }
+scale-info = { version = "2.1.1", default-features = false, features = [
+	"derive",
+] }
+
+# util
+log = { version = "0.4.17", default-features = false }
```

Comment:
This is no longer used, it was used at one point in the migrations.rs module which has now been removed.